contracts/utils/RSKAddrValidator.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.2;
+
+library RSKAddrValidator {
+
+    /*
+    * @param addr it is an address to check that it does not originates from 
+    * signing with PK = ZERO. RSK has a small difference in which @ZERO_PK_ADDR is 
+    * also an address from PK = ZERO. So we check for both of them.
+    */
+    function checkPKNotZero(address addr) internal pure returns (bool){
+        return (addr != address(0) && addr != 0xdcc703c0E500B653Ca82273B7BFAd8045D85a470);
```

Comment:
given that RSK does not return address(0), it is better to put the 0xdcc703c0E500B653Ca82273B7BFAd8045D85a470 check first, the address(0) is there just for compatibility purposes and makes no sense that it is first in the "ifthen" check, it makes it more expensive in the case where the address was, in fact, the RSK's 0 version (0xdcc703c0E500B653Ca82273B7BFAd8045D85a470).